Source/CarthageKit/Xcode.swift

```diff
@@ -332,7 +332,7 @@ public enum SDK: String {
 
 	/// Attempts to parse an SDK name from a string returned from `xcodebuild`.
 	public static func fromString(string: String) -> Result<SDK, CarthageError> {
-		return Result(self(rawValue: string), failWith: .ParseError(description: "unexpected SDK key \"(string)\""))
+		return Result(self(rawValue: string), failWith: .ParseError(description: "unexpected SDK key \(string)"))
```

Comment:
I think this intends to quote the `string` value, so we should fix as `"unexpected SDK key \"\(string)\""`.